docs/utils/iers.rst

```diff
@@ -70,28 +74,9 @@ and includes transforms dating back to 1973-01-01.
 Configuration parameters
 ------------------------
 
-There are three configuration parameters that control the behavior
```

Comment:
Even though I see the point of single-sourcing these definitions, I think it is useful in the narrative docs to repeat them. The whole behavior of IERS is a bit tricky and so being able to see the definitions in the immediate context of the discussions below is helpful.

Reply:
But the original text only listed a subset, so is the intention to only list what you think is important, or everything? It would risk going out of sync again with reality.

Reply:
I looked in the docs and the original three (auto_download, auto_max_age, and remote_timeout) are the ones that get referred to later in the document. These three are the most likely to get updated by users. So I would go with a hybrid version:
```
There are a number of `IERS configuration parameters <link>`_ that control the 
behavior of the automatic IERS downloading. Three of the most important to consider
are the following::
```
The `<link>`  should get the user to https://docs.astropy.org/en/latest/api/astropy.utils.iers.Conf.html#astropy.utils.iers.Conf.

Reply:
Updated as you suggested. Thanks!